prac_07/project.py

```diff
@@ -0,0 +1,19 @@
+import datetime
```

Comment:
is date time needed in the class, if used properly in the other file its probably just more typing in the class than neccessary